### DISCUSSION TITLE
Yaml Localization

### DISCUSSION THREAD

**Discussion by subzero911** [jonataslaw/getx#1072] [Ideas]
Please add yaml support (as in https://pub.dev/packages/easy_localization).
Also add multiple yaml files support (like 'profile.en.yaml', 'home.en.yaml', 'login.en.yaml'), because single file easily gets bloated.

Your current syntax:
![image](https://user-images.githubusercontent.com/12999702/107847621-2ed89a00-6dfe-11eb-87f2-7e6434e48238.png)
Requires much manual work and impossible to structure data when it comes to thousands of keys. It would lead to 'reminderDetailProductSelectorDescriptionText' key names.

Yaml
![image](https://user-images.githubusercontent.com/12999702/107847576-d0abb700-6dfd-11eb-8ba4-4e7f452748cc.png)
Much more concise and readable syntax, rich formatting features (multi-line text etc.)

**Comment by jonataslaw** [comment]
Moving that to discussions.
It may be possible and easier to do this with get_cli than with the package itself, but I don't rule that out.
